src/index.html

```diff
@@ -5,9 +5,246 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="./images/Icon-Corona.svg">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <div class="header__nav">
+        <div class="header__nav-burger">
+          <input
+            class="header__nav-checkbox"
+            type="checkbox"
+            id="menu"
+          >
+          <label class="header__nav-label" for="menu"></label>
+          <nav class="header__nav-menu">
+            <a class="header__nav-link" href="#home">HOME</a>
+            <a class="header__nav-link" href="#about">ABOUT US</a>
+            <a class="header__nav-link" href="#gallery">GALLERY</a>
+            <a class="header__nav-link" href="#consultation">CONSULTATION</a>
+            <a class="header__nav-link" href="#testimonials">TESTIMONIALS</a>
+            <a class="header__nav-link" href="#contacts">CONTACTS</a>
+            <a href="tel:+1 234 555-55-55" class="header__nav-contact header__nav-contact--number">+1 234 555-55-55</a>
+            <p class="header__nav-contact header__nav-contact--title">CALL TO ORDER</p>
+          </nav>
+        </div>
+
+        <a href="#" class="header__nav-logo">
+          <img
+            class="header__nav-logo"
+            src="/images/logo1.svg"
+            alt="Logo"
+          >
+        </a>
+        <div class="header__nav-phone">
+          <a href="tel:+1 234 555-55-55" class="header__nav-phone header__nav-phone--number">+1 234 555-55-55</a>
+          <a href="tel:+1 234 555-55-55">
+            <img class="header__nav-img" src="/images/phone.svg"
+              alt="Contact">
+          </a>
+        </div>
+      </div>
+      <h1 class="header__logo">Miami Condo Kings</h1>
+    </header>
+
+    <main class="main main--container">
```

Comment:
```suggestion
    <main class="main main__container">
```
It's an element (not modifier)